ember-headlessui/addon/components/tabs-group.js

```diff
@@ -0,0 +1,36 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+export default class TABSGROUP extends Component {
+  tabNames = [];
+  Contents = [];
+  @tracked currentTab = null;
+  @action
+  registerTabNames(e) {
+    this.tabNames = [...this.tabNames, e];
```

Comment:
instead of using immutability patterns, can we switch to a `TrackedArray`? that will provide more optimal updates and not invalidate the whole collection